ducktape/__main__.py

```diff
@@ -0,0 +1,131 @@
+from ducktape.tests.test import TestLoader
```

Comment:
I was referring to the second half of the first rule. Quoting the PEP:

> It's okay to say this though:
> 
> `from subprocess import Popen, PIPE`

So I was just saying it might be nicer to have

```
from ducktape.tests.test import TestLoader, SerialTestRunner, SimpleStdoutReport, SimpleFileReporter, TestSessionContext
```

In any case, it's not a big deal, unless we adopt a standard format, different people's IDEs are probably going to end up formatting stuff like this in a variety of ways.